langchain/src/vectorstores/tests/vectara.int.test.ts

```diff
@@ -154,5 +156,41 @@ describe("VectaraStore", () => {
       );
       expect(hasEnglish).toBe(false);
     });
+
+    it("addFiles", async () => {
+      const docs = getDocs();
+      const englishOneContent = docs[0].pageContent;
+      const englishTwoContent = docs[1].pageContent;
+      const frenchOneContent = docs[2].pageContent;
+
+      // Create temporary files for test
+      const files = [
+        { filename: "englishOne.txt", content: englishOneContent },
+        { filename: "englishTwo.txt", content: englishTwoContent },
+        { filename: "frenchOne.txt", content: frenchOneContent },
+      ];
+
+      // Using async/await and the helper function to create files
+      for (const file of files) {
+        fs.writeFile(file.filename, file.content, (err) => {
+          if (err) throw err;
+        });
+      }
+
+      const results = await store.addFiles([
+        "./englishOne.txt",
```

Comment:
Any way we can do also a test with a small PDF file instead of just text?

Reply:
Yes